test/e2e/network/service.go

```diff
@@ -3175,6 +3175,18 @@ func execAffinityTestForSessionAffinityTimeout(f *framework.Framework, cs client
 	serviceType := svc.Spec.Type
 	// set an affinity timeout equal to the number of connection requests
 	svcSessionAffinityTimeout := int32(AffinityConfirmCount)
+	if proxyMode, err := proxyMode(f); err == nil {
+		if proxyMode == "ipvs" {
+			// session affinity timeout must be bigger than 120 in ipvs mode,
+			// because IPVS module has a hardcoded TIME_WAIT timeout of 120s,
+			// and that value can't be sysctl'ed now.
+			// Ref: https://github.com/torvalds/linux/blob/master/net/netfilter/ipvs/ip_vs_proto_tcp.c
+			// TODO: remove this to speed up testing when IPVS does really respect session affinity timeout
+			svcSessionAffinityTimeout = int32(135)
```

Comment:
I'm not sure that increasing the session affinity timeout fixes the problem because it's 120s in addition to the persistent timer set on the IPVS virtual server. No matter what session affinity timeout is set, IPVS will always do timeout + 120s. Let's run tests to find out though :)

Reply:
Also I feel like this is just working around the actual problem, would prefer to see the tests failing until there is a proper fix.

Reply:
What @aojea said :)